packages/website/docs/docs/nextjs.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Next.js and BlockNote
+description: Details on integrating BlockNote with Next.js
+imageTitle: Next.js and BlockNote
+path: /docs/nextjs
+---
+
+# Next.js and BlockNote
+
+(only for next.js users)
```

Comment:
Don't think this line is necessary, the page is pretty clearly already targeted at NextJS users

Reply:
agree, rephrased